src/mlpack/methods/kde/kde_rules_impl.hpp

```diff
@@ -115,7 +115,8 @@ Score(const size_t queryIndex, TreeType& referenceNode)
     // Don't duplicate calculations.
     alreadyDidRefPoint0 = true;
     const double furthestDescDist = referenceNode.FurthestDescendantDistance();
-    minDistance = traversalInfo.LastBaseCase() - furthestDescDist;
+    minDistance = std::max(traversalInfo.LastBaseCase() - furthestDescDist,
+        0.0);
```

Comment:
Super minor style question here, should this be lined up with the previous argument `traversalInfo.LastBaseCase() - furthestDescDist` or does this just apply to method declarations?